docs/wrappers/Magento

```diff
@@ -0,0 +1,14 @@
+[---
```

Comment:
This file doesn't seem to be a valid markdown file. How did you test this? The filename doesn't have `.md` extension, too.

Reply:
Hi sorry i realise i made some mistake while moving it , i've made a different pull request here,
thank you,